projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.scss

```diff
@@ -0,0 +1,44 @@
+.jetpack-email-preview {
+  height: 200px;
+  width: 645px;
+  left: 391px;
+  top: 392px;
```

Comment:
How does the modal behave without defining these sizes/positions?

If possible without any overrides is best.

We could do `max-width` and `max-height` if we must.